pkg/limayaml/default.yaml

```diff
@@ -31,6 +31,19 @@ memory: "4GiB"
 # Default: "100GiB"
 disk: "100GiB"
 
+network:
+  # The instance can get a routable IP address from the vmnet framework using
+  # https://github.com/AkihiroSuda/vde_vmnet. Both vde_switch and vde_vmnet
+  # daemons must be running before the instance is started. The interface type
+  # (host, shared, or bridged) is configured in vde_vmnet and not lima.
+  vde:
+    # vde_switch socket directory, normally /var/run/vde.ctl
+    # Default: ~ (vmnet is not being used)
```

Comment:
The default should be empty.

~ means home directory.

Reply:
I assume you are just talking about the comment, and want it to look like this:

```
    # Default: "" (vmnet is not being used)
```

Do you want the YAML values changed as well? Because there `~` means `null` and not `"~"`.

Reply:
Aha, didn’t know that… thanks for the info.

 I prefer to use empty string, as ~ really looks like home directory.

Reply:
I've put `null` for the value, but `""` in the comment. I can put `""` as the values as well, as that's what they'll eventually end up being anyways.